pylean/pylean/__init__.py

```diff
@@ -0,0 +1,37 @@
+import pexpect
+import json
+
+import os
+
+class LeanServer:
+    def __init__(self):
+        path_to_repl = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
+
+        self.proc = pexpect.spawn(
+            "lake env lean --run REPL/Main.lean", cwd=path_to_repl, encoding="utf-8"
+        )
+
+    def run_code(self, code, env=None, verbose=False):
+        if env:
+            command = (
+                '{ "cmd" : "' + repr(code)[1:-1] + f'", "env" : {env}' + " }"
```

Comment:
Am I right in saying this is json? If so, can you build it using the `json` module to ensure things are correctly escaped?

Reply:
```suggestion
                json.dumps(dict(cmd=code, env=env))
```

Reply:
You're right. For some reason, I assumed that the spacing between brackets and quotes in the README was necessary, but that's clearly not the case.